n2y/converter.py

```diff
@@ -46,9 +47,11 @@ def load_plugins(filename):
                     and issubclass(class_to_replace, Block):
                 globals()[key] = value
             else:
-                print(f"Cannot import plugin \"{key}\" since it not derrived from a known class.")
+                logger.warning("%s %s" % (
```

Comment:
I don't think you should need to use the `%` operator here. Instead, you can use `logger.warning("%s %s",  f"Warning: Cannot Import Plugin \"{key}\"", "Because It Is Not Derrived From A Known Class.")`

Or, even better, you can use:

`logger.warning('Warning: Cannot import plugin "%s" because it is not derived from a known class.', key)`

This is better for a few reasons:

1. There's no need to use %s formatting for string literals; you can just combine the strings directly.
2. It avoids using f-strings in the log messages; this is good because it avoids wasting processing time formatting the string if the logging settings are set such that the message won't be formatted. By using the %s syntax for the key, the logging module can be smart enough to skip bothering with this if the message would just get thrown out anyway.
3. It doesn't capitalize all of the words in the message (usually you would write messages as if they're a sentence in regular English.